src/lib/y2storage/autoinst_profile/partitioning_section.rb

```diff
@@ -83,7 +83,7 @@ def self.new_from_hashes(drives_array)
       def self.new_from_storage(devicegraph)
         result = new
         # TODO: consider also NFS and TMPFS
-        devices = devicegraph.md_raids + devicegraph.lvm_vgs + devicegraph.disk_devices
+        devices = devicegraph.software_raids + devicegraph.lvm_vgs + devicegraph.disk_devices
```

Comment:
So the MD devices are not handled any more? Or are they covered by the `software_raids`?

Reply:
This is covered by the class hierarchy AFAICS.